extensions/transactions/create-replace-update-delete/standard/requirements/create-replace-delete/REQ_methods.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_core_methods]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:per-id}* |*/req/core/methods*
+^|A |A server SHALL implement one or more of the methods HTTP POST, PUT and/or DELETE for each mutable resource.
+^|B |A server SHALL declare which methods are supported for each mutable resource via the <<options,HTTP OPTIONS>> method.
```

Comment:
Sub-requirement B makes the Options requirements class mandatory whenever Create/Replace/Delete is implemented. Which means that the requirements class is not necessary and all provisions can be moved to the Create/Replace/Delete requirements class. 

Either this, or remove B. However, in that case, the client needs to investigate the OpenAPI definition to find out if POST, PUT and/or DELETE is supported.

Reply:
Meeting 2022-11-21: Move the OPTIONS text into Clause 6 and the Create, Replace, Delete requirements class.

Reply:
Removed OPTIONS requirements class and merged it into clause 6.